vars/main.yml

```diff
@@ -39,6 +39,11 @@ jenkins_plugins_dir: "{{ jenkins_home }}/plugins"
 # Directory for secret files (credentials, etc)
 jenkins_secrets_dir: "{{ jenkins_root }}/secrets"
 
+# Name of the Jenkins service, as listed by the actual service manager. This is only used
```

Comment:
> as listed by the actual service manager

That's only true when the service manager is systemd, so that sounds weird.

Reply:
Fixed in b674282.